frontend/src/components/about/Platform.vue

```diff
@@ -49,13 +49,20 @@
       </p>
 
       <h3 id="citation" class="title is-3 pt-6">Citation</h3>
-
-      <h4 class="title is-4">Latest publication</h4>
       <p>
         If you use Metabolic Atlas in your work, or any of the animal models, please cite our latest
         publication. In addition, if you use Human-GEM, please also cite the corresponding
```

Comment:
I might be good to add a link for the `corresponding publication` for Human-GEM.

Reply:
True! Added a link in this commit 408c7a2